app/src/main/java/com/nextcloud/talk/messagesearch/MessageSearchActivity.kt

```diff
@@ -2,8 +2,10 @@
  * Nextcloud Talk application
  *
  * @author Álvaro Brey
+ * @author Ezhil Shanmugham
```

Comment:
no changes in this file, so no authorship, probably a copy&paste thing

Reply:
...or it is lacking the back-pressed changes

Reply:
I mistakenly added authorship while adding for the rest